gnomad_qc/v5/annotations/generate_variant_qc_annotations.py

```diff
@@ -0,0 +1,135 @@
+"""Script to generate annotations for variant QC on gnomAD v5."""
+
+import argparse
+import logging
+
+import hail as hl
+from gnomad.variant_qc.pipeline import generate_sib_stats, generate_trio_stats
+
+from gnomad_qc.resource_utils import check_resource_existence
+from gnomad_qc.v5.annotations.annotation_utils import annotate_adj
+from gnomad_qc.v5.resources.annotations import get_sib_stats, get_trio_stats
+from gnomad_qc.v5.resources.basics import (
+    WORKSPACE_BUCKET,
+    get_aou_vds,
+    get_logging_path,
+)
+from gnomad_qc.v5.resources.constants import GNOMAD_TMP_BUCKET
+from gnomad_qc.v5.resources.sample_qc import dense_trios, pedigree, relatedness
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("generate_variant_qc_annotations")
+logger.setLevel(logging.INFO)
+
+
+def run_generate_trio_stats(
+    mt: hl.MatrixTable,
+    fam_ped: hl.Pedigree,
+) -> hl.Table:
+    """
+    Generate trio transmission stats from a VariantDataset and pedigree info.
+
+    :param mt: Dense trio MatrixTable.
+    :param fam_ped: Pedigree containing trio info.
+    :return: Table containing trio stats.
+    """
+    # Add adj annotation and convert LGT to GT since only
+    # autosomal bi-allelics are used to calculate trio stats.
+    mt = annotate_adj(mt)
+    mt = mt.transmute_entries(GT=mt.LGT)
```

Comment:
Should this just move to the create_dense_trio_mt function in identify trios? I can see not storing adj there but since its biallelic already, maybe just update to GT there.